src/main/java/org/betonquest/betonquest/item/QuestItemHandler.java

```diff
@@ -309,8 +309,4 @@ public void onPlayerSwapHandItems(final PlayerSwapHandItemsEvent event) {
     private boolean isJournalSlotLocked() {
         return Boolean.parseBoolean(Config.getConfigString("journal.lock_default_journal_slot"));
     }
-
-    public HandlerList getHandlers() {
-        return HANDLERS;
-    }
```

Comment:
I am sure this needs to be implemented. Did you run it on a server?

Reply:
This is part of the Bukkit Event system to register listener on it.
It was added in 40e009d112d456b7dd64dfa820393a6c833fcf56 without any reason I can see.